src/main/java/com/conveyal/gtfs/GTFSCache.java

```diff
@@ -96,14 +74,18 @@ private void createCache(int cacheSize) {
             }
         };
         this.cache = CacheBuilder.newBuilder()
-            .maximumSize(cacheSize)
-            .removalListener(removalListener)
-            .build(new CacheLoader<String, GTFSFeed>() {
-                @Override
-                public GTFSFeed load(String s) throws Exception {
-                    return retrieveFeed(s);
-                }
-            });
+                .maximumSize(cacheSize)
+                .removalListener(removalListener)
+                .build(new CacheLoader<String, GTFSFeed>() {
+                    @Override
+                    public GTFSFeed load(String s) throws Exception {
+                        return retrieveFeed(s);
+                    }
+                });
+    }
+
+    public long getCurrentCacheSize() {
+        return this.cache.size();
```

Comment:
Did you intend to have this return the max size? I think that this will return the number of items currently in the cache (which will be 0 when the server first starts up).